cmd/notation/verify.go

```diff
@@ -148,3 +160,20 @@ func resolveReference(ctx context.Context, opts *SecureFlagOpts, reference strin
 
 	return ref, nil
 }
+
+func printMetadataIfPresent(outcome *notation.VerificationOutcome) {
+	// ignoring error, getting metadata for a successful outcome
+	metadata, _ := outcome.GetUserMetadata()
```

Comment:
Why are we ignoring error here? We are not calling `GetUserMetadata` anywhere in verification so a code bug can still lead to a error.